src/routes/controllers/AuthenticationControllers/signInUser.ts

```diff
@@ -28,6 +29,31 @@ export const signInUser = async (req: Request, res: Response) => {
       },
     };
 
+    var message : String;
+
+    if (userRole == "chef" || userRole == "delivery") {
+      const employee: Employees | null = await EmployeesModel.findOne({
```

Comment:
you don't have to call findOne again you can just use user object cuz at this point user will either be a chef or delivery